code/frameworks/vue3-vite/package.json

```diff
@@ -79,6 +79,11 @@
       "./src/index.ts",
       "./src/preset.ts"
     ],
+    "externals": [
+      "util",
+      "tty",
+      "path"
```

Comment:
Why do these need to be listed as externals now?

Reply:
Good question.. I do not remember 2 weeks later..
I think something relating to the package being bundled in using this, but them being node-internals?